public/js/create-recipe.js

```diff
@@ -1 +1,59 @@
-// Create Recipe Form code goes here
\ No newline at end of file
+// Create Recipe Form code goes here
+
+// create-recipe.handlebars should contain
+// Title
+// Ingredients List
+// Input box to search for ingredients
+// Submit ingredient button to initiate search for ingredients
+// For each ingredient 2 more input boxes should be created, 1 for measure and 1 for quantity
+// Method Textarea
+// Add Images Option
+// Final submit recipe button
+
+// Title
+
+// Ingredients List
+// Search box to input ingredient
+// Users enter ingredient name into the search box
+// Submit button
+// User clicks the submit button
+
+// Check local database if ingredient already exists
+// If exists then add to ingredients list
+// If not then make the API call
+// 
+// API Call
+// If ingredient is found
+// Get price and add to ingredients list
+// If not found
+// Ask user to input ingredient manually with price and category (category optional)
+
+// Household ingredients
+// If ingredient is household (salt, herbs, spices, etc...)
+// No need for price
+// Everything else needs a price
+
+// Method
+// Textarea
+
+// Images
+// How to add images to the database?
+
+// Submit the recipe
+// 
+// To go into the ingredients database
+// Name
+// Price
+// Category?
+// 
+// To go into the recipe database
+// Title
+// Method
+// Images
+// 
+// To go into the measures database if measure does not exist
+// Measure name
```

Comment:
the measures could just be a dropdown box with a list of hardcoded units (eg, cups)